rest-service/manager_rest/resource_manager.py

```diff
@@ -145,8 +145,6 @@ def _get_conf_for_snapshots_wf():
         config_instance = config.instance
         return {
             'file_server_root': config_instance.file_server_root,
-            'db_address': config_instance.db_address,
```

Comment:
Why do we not need to return these here any more?

Reply:
because they're not used

Reply:
Fair enough, approved then. <smiles>